bodhi/client/__init__.py

```diff
@@ -301,9 +303,19 @@ def query(url, **kwargs):
     Args:
         url (unicode): The URL of a Bodhi server to create the update on. Ignored if staging is
                        True.
+        mine (Boolean): If the --mine flag was set
         kwargs (dict): Other keyword arguments passed to us by click.
     """
     client = bindings.BodhiClient(base_url=url, staging=kwargs['staging'])
+    if mine:
+        if 'user' in kwargs:
```

Comment:
I think this ```if``` statement might always be True. In my testing, I neither set the environment variable nor set the --user flag, but saw:

```
Warning: --mine and --user options both set. --user will be ignored.
```

In these cases, ```user``` is in ```kwargs``` but is set to ```None```. Perhaps the ```if``` statement here and elsewhere should be:

```
if kwargs.get('user', None):
```